pkg/ccn-lite/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=ccn-lite
 PKG_URL=https://github.com/cn-uofbasel/ccn-lite/
-PKG_VERSION=0b1de2da1ef407ee5793c0e7eda420391ae056dc
+PKG_VERSION=3bddb34dbb27d814acaca922655d760481b2998a
```

Comment:
Please update to https://github.com/cn-uofbasel/ccn-lite/commit/10119ca8173457e8a26b28fc9f30fe0d97d17857